### PR TITLE
Revamp README

### PR DESCRIPTION
**Pull request by azigler**
Okay, I reworked the README to be more actionable for a community member:

- Expanded README to match standard format
- Simplified install instructions to work with `make deploy` locally after auto-creating the admin, team, and channel
- Added additional links

Should be pretty straightforward to take for a spin locally, let me know if you hit any issues!

@crspeller @chenilim

### CODE REVIEW

File: mattermost/.docker.env
Comment (crspeller) [inline mattermost/.docker.env:1]: Accidentally committed this empty folder and file?

File: README.md [hunk 1]
```diff
@@ -1,43 +1,99 @@
...
-## Installation
+Explore Mattermost's AI initiatives:
 
-1. Go the releases page and download the latest release.
-2. On your Mattermost, go to System Console -> Plugin Management and upload it. [More Details](https://docs.mattermost.com/administration/plugins.html#plugin-uploads)
-3. Enable the plugin and configure plugin settings as desired.
+* https://mattermost.github.io/mattermost-ai-site/
+* https://community.mattermost.com/core/channels/ai-exchange
+* https://forum.mattermost.com/c/ai-frameworks/40
+* https://mattermost.github.io/mattermost-ai-framework/
```
Comment (crspeller) [inline README.md]: Old URL. Dead?

File: README.md [hunk 2]
```diff
@@ -1,43 +1,99 @@
...
 
-## Installation
+Explore Mattermost's AI initiatives:
 
-1. Go the releases page and download the latest release.
-2. On your Mattermost, go to System Console -> Plugin Management and upload it. [More Details](https://docs.mattermost.com/administration/plugins.html#plugin-uploads)
-3. Enable the plugin and configure plugin settings as desired.
+* https://mattermost.github.io/mattermost-ai-site/
+* https://community.mattermost.com/core/channels/ai-exchange
+* https://forum.mattermost.com/c/ai-frameworks/40
```
Comment (crspeller) [inline README.md:88]: We could enable discussions on this repo instead? What do you think?

File: README.md [hunk 3]
```diff
@@ -1,43 +1,99 @@
...
+1. Clone and enter this repository:
+  * `git clone https://github.com/mattermost/mattermost-plugin-ai && cd mattermost-plugin-ai`
+2. Start the services: `docker compose up -d`
+3. Configure the Mattermost server from the init script: `bash ./init.sh`
+4. Access Mattermost:
+  * Open Mattermost at `http://localhost:8065`
+  * Select **View in Browser**
+  * Log in with the generated `root` credentials
+5. Install the `mattermost-plugin-ai` on Mattermost:
+  * Go the releases page and download the latest release.
```
Comment (crspeller) [inline README.md]: Don't actually have releases yet. I can add one if you would like. Or maybe just run `make deploy` for the quick setup for now?

File: README.md [hunk 4]
```diff
@@ -1,43 +1,99 @@
...
+**🚀 Check out [our AI developer website](https://mattermost.github.io/mattermost-ai-site/) and join the ["AI Exchange" channel](https://community.mattermost.com/core/channels/ai-exchange) where Mattermost's open source community is sharing AI news and innovation in real time!**
 
 The LLM extensions plugin adds functionality around the use and development of LLMs like GPT-3.5 / 4 and hugging face models within Mattermost. 
 
 Currently at the experimental phase of development. Contributions and suggestions welcome! 
 
-## Conversation
+## Install
+
+1. Clone and enter this repository:
```
Comment (crspeller) [inline README.md:34]: This really just setts up a MM server. Maybe we should call that out? If you already have one try it there instead. 

I was going to say that we could just link to the server README for this but it seems we removed that at some point. There is still a dead link in the TOC: https://github.com/mattermost/mattermost-server#table-of-contents

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+      MM_SQLSETTINGS_DATASOURCE: "postgres://mmuser:mostest@db:5432/mattermost_test?sslmode=disable\u0026connect_timeout=10"
+      MM_SERVICESETTINGS_LISTENADDRESS: ":8065"
+      MM_SERVICESETTINGS_SITEURL: "http://localhost:8065"
+      MM_SERVICESETTINGS_ENABLEBOTACCOUNTCREATION: "true"
+      MM_SERVICESETTINGS_ENABLEUSERACCESSTOKENS: "true"
+      MM_SERVICESETTINGS_ENABLEOAUTHSERVICEPROVIDER: "true"
+      MM_SERVICESETTINGsS_ENABLEDEVELOPER: "true"
+      MM_SERVICESETTINGS_ENABLETESTING: "true"
+      MM_PLUGINSETTINGS_AUTOMATICPREPACKAGEDPLUGINS: "true"
+      MM_EXPERIMENTALSETTINGS_ENABLEAPPBAR: "true"
```
Comment (crspeller) [inline docker-compose.yml]: Some of these seem unnecessary. Why are we enabling the apps bar?

File: docker-compose.yml
Comment (crspeller) [inline docker-compose.yml:1]: could we put the docker stuff into a docker sub-folder?

**Review by crspeller — CHANGES_REQUESTED** [review]
Awesome! I love the standard README.

File: README.md (hunk 2 above)
Comment (azigler) [reply README.md:88]: Definitely, I think we can just swap out these links in the coming days as we organize our content.

File: README.md (hunk 3 above)
Comment (azigler) [reply README.md]: Oops, looks like my final README version with `make deploy` didn't make the PR. I just pushed it now.

File: docker-compose.yml
Comment (azigler) [reply docker-compose.yml:1]: Then we have to tell the user to also go in that folder to run `docker compose up` so I think it's easier to keep it in root, 1/5.

File: README.md (hunk 4 above)
Comment (azigler) [reply README.md:34]: I added a note in the install header to clarify!

**Comment by azigler** [comment]
Thanks for the review @crspeller, I incorporated those changes.

**Review by crspeller — APPROVED** [review]
Great!

**Pull request merged**